### ISSUE TITLE
PascalCase for type names, or interfaces instead of types in modesl

### ISSUE THREAD

**Issue by IamIGI** [ferdikoomen/openapi-typescript-codegen#1893]
Like in the title.
Is there an option to set, that will change  the name of models from camelCase to PascalCase, and option that would set model to be declared as interfaces instead of types ?

By default we have types with camelCase names
![image](https://github.com/ferdikoomen/openapi-typescript-codegen/assets/84968638/6d292ad7-2598-45b4-8569-7b9b73da3d9a)
I would like to get intefaces with PascalCase names.
![image](https://github.com/ferdikoomen/openapi-typescript-codegen/assets/84968638/571afc41-140c-4935-841b-6858ef6258cd)

**Comment by pawKer** [comment]
+1 also interested to know if there's a way to do this without actually changing the types in the OpenAPI file to PascalCase.

**Comment by mrlubos** [comment]
Yep, we will add this option in https://github.com/hey-api/openapi-ts
